submissions/anishchenko/html-css-popup/css/style.css

```diff
@@ -0,0 +1,322 @@
+* {
+  box-sizing: border-box;
+}
+
+img {
+  width: 100%;
+  height: auto;
+}
+
+a {
+  display: block;
+  text-decoration: none;
+}
+
+.visualy_hidden:not(:focus):not(:active),
+input[type="checkbox"].visualy_hidden,
+input[type="radio"].visualy_hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  white-space: nowrap;
+  clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+header {
+  margin-top: 0.8rem;
+}
+
+main {
```

Comment:
You have classes `header` and `main`, why you don't use them in css?🙂

Reply:
i hope that i fix this problem)) thanks for review